src/renderer/extensions/vueNodes/components/LGraphNode.vue

```diff
@@ -191,6 +191,17 @@ const isSelected = computed(() => {
   return selectedNodeIds.value.has(props.nodeData.id)
 })
 
+// Execution error via DI (single current node id)
+const execErrorNodeId = inject(ExecutionErrorNodeIdKey, undefined)
+const hasExecutionError = computed(
+  () => execErrorNodeId?.value === props.nodeData.id
+)
+
+// Computed error states for styling
+const hasAnyError = computed(
+  () => hasExecutionError.value || props.nodeData.hasErrors
```

Comment:
To protect against the one bug where we missed a `.value`
```suggestion
  (): boolean => hasExecutionError.value || props.nodeData.hasErrors
```